### PR TITLE
#1242 Fix issue with incorrect applicants moved to withdrawn status

### PR DESCRIPTION
**Pull request by mbrookeswebdev**

### CODE REVIEW

**Comment by mbrookeswebdev** [comment]
To test:
Please try moving several candidates from stage Shortlisted to stage Selected (Invited To Selection Day) and then a few other candidates to Withdrawn. Only correct candidates should be moved to Withdrawn.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "admin",
-  "version": "0.59.0",
+  "version": "0.59.1",
```
Comment (tomlovesgithub) [inline package-lock.json:3]: is this going to be released as a hotfix?

**Review by tomlovesgithub — APPROVED** [review]
Looks right, just wondering how it should be released (hotfix/otherwise)

File: package-lock.json (hunk 1 above)
Comment (mbrookeswebdev) [reply package-lock.json:3]: In the platform-issues form it says it was not urgent.

**Review by warrensearle — APPROVED** [review]
Tested using code on `develop` and again with this code and can see that it fixes the problem 👍

**Pull request merged**